### PR TITLE
Set up Github Actions

### PR DESCRIPTION
**Pull request by mgax**
* Update the Readme with syntax highlighting.
* Update PyPI links.
* Set up CI and publishing actions.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,10 +1,17 @@
 [tox]
 envlist =
-  py39-django42-wagtail62
+  py{39,310,311,312}-django42-wagtail62
   py{310,311,312}-django{50,51}-wagtail62
   py312-django42-wagtail{52,60,61,62}
```
Comment (zerolab) [inline tox.ini:5]:
```suggestion
  py312-django42-wagtail{52,62}
```

since 6.0 and 6.1 are no longer officially supported

Comment (mgax) [reply tox.ini]: Ah, good call about Wagtail versions, thanks!

I'm not sure about only supporting Python 3.11+ though. In general, I agree with the sentiment, but I can imagine somebody jumping in to rescue a legacy project, and hoping to pull the production database, without doing a major upgrade beforehand. It's a simple package, and doesn't cost us much to support older Pythons (aside from some CI time).

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  nightly-wagtail-test:
+    runs-on: ubuntu-latest
+    env:
+      WEBHOOK_EXISTS: ${{ secrets.SLACK_WEBHOOK_URL != '' }}
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
```
Comment (zerolab) [inline .github/workflows/nightly.yml:19]: suggestion for follow-up PR: use the latest well tested version. In this case 3.12

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.9'
```
Comment (zerolab) [inline .github/workflows/test.yml:28]: suggestion for follow up PR: contrary to popular belief, it is best to use the latest stable/tested version (so 3.12) to benefit from improvements. All current tooling will run on a newer version but do the linting for prev versions

File: pyproject.toml [hunk 4]
```diff
@@ -23,7 +24,7 @@ dependencies = [
 requires-python = ">=3.9"
 
 [project.urls]
-Home = "https://git.torchbox.com/internal/django-birdbath"
+Home = "https://github.com/torchbox/django-birdbath"
```
Comment (zerolab) [inline pyproject.toml:27]: suggestion: add Django >= 4.2 to the dependencies list.
This is, after all, django-birdbath ;)

File: tox.ini [hunk 5]
```diff
@@ -1,18 +1,24 @@
 [tox]
 envlist =
-  py39-django42-wagtail62
-  py{310,311,312}-django{50,51}-wagtail62
-  py312-django42-wagtail{52,60,61,62}
+  py{39,310,311,312,313}-django42-wagtail62
+  py{310,311,312,313}-django{50,51}-wagtail62
+  py{313}-django42-wagtail{52,62}
```
Comment (zerolab) [inline tox.ini:5]: non-blocking: best to do Watail LTS with the Django LTS (so `-django42-wagtail{52,62}`. This also works, just that neither Django 4.2, nor Wagtail 5.2 declare official support for Python 3.13

**Review by zerolab — APPROVED** [review]
Added a few suggestions that can be addressed as a follow up.
Thanks @mgax

File: .github/workflows/test.yml (hunk 3 above)
Comment (mgax) [reply .github/workflows/test.yml]: That makes more sense, thanks for explaining it.

File: pyproject.toml (hunk 4 above)
Comment (mgax) [reply pyproject.toml:28]: Hah, good point!

File: tox.ini (hunk 5 above)
Comment (mgax) [reply tox.ini]: Hmm, I wonder how py3.13 got in there. I've messed with the matrix too much and lost track of things. So I've added a couple of comments to explain what each section should do. Thanks for flagging this!

**Pull request merged**